### ISSUE TITLE
[Component name] Describe in detail how to use display cutouts in landscape mode in different scenarios and with each components. 

### ISSUE THREAD

**Issue by nailsadEleos** [material-components/material-components-android#3870]
**Is your feature request related to a problem? Please describe.**
Most app developers don't know what to do with the display cutout in landscape mode, as a result, it remains unused and has a black color. And it's not surprising, since even on the Material IO website there are literally a couple of paragraphs (https://m3.material.io/foundations/layout/understanding-layout/hardware-considerations#05da90ed-7115-4c32-83c8-3164e00c6dfc) devoted to it without any instructions for each component. 

This approach breaks the immersive experience and causes people to use apps less in landscape orientation. 

**Describe the solution you'd like**
I'd like you to describe how to use or nicely design a display cutout for apps in landscape orientation for each component. Speaking of components I mean navigation rails, top bars, bottom bars, bottom sheets and so on. 

I would also suggest making it so that all components and templates support the use of the display cutout by default. 

As a solution, I suggest you keep the padding of the various controls, but extend the components' backerboards to the cutout area. 

**Additional context**
In the attached screenshots, I have given examples from Google apps, where in 90% of cases (and not only Google apps) the display cutout is left unused, causing it to be black. In the same screenshots, I have given the desired result of how the cutout should look like in landscape orientation. 

I would say that among all Google apps, Google News has the best implementation of landscape orientation using a display cutout. 
![Gboard landscape display cutout issue](https://github.com/material-components/material-components-android/assets/103059958/4e61a869-5221-4771-824a-db5ef1c3f503)
![Gboard settings landscape display cutout issue](https://github.com/material-components/material-components-android/assets/103059958/9c5b2067-fe34-4090-90d0-4ce7845f2241)
![Gboard settings landscape display cutout issue2](https://github.com/material-components/material-components-android/assets/103059958/9434fba5-a211-4354-bc04-ecc044186d8b)
![Google Messages landscape mode issue](https://github.com/material-components/material-components-android/assets/103059958/626a60f4-40b8-47eb-a1ec-53bbac1c9ad6)
![Keep display cutout issue in landscape mode](https://github.com/material-components/material-components-android/assets/103059958/3d1352cc-9347-4e37-89b7-795dd140b553)
![Google News (best example of how the display cutout in album mode should be implemented)](https://github.com/material-components/material-components-android/assets/103059958/a52f72d6-0cd9-420a-bade-ac4a41a992de)
![Google_Weather_Main_page__display_cutout_on_the_left](https://github.com/material-components/material-components-android/assets/103059958/c5767cfe-0058-4420-8f5c-a4430d7e04f9)
![Google_Weather_Main_page__display_cutout_on_the_right](https://github.com/material-components/material-components-android/assets/103059958/5aa577df-1feb-4d05-ae1b-46d4add52be0)
![Google_Weather_Search_page__display_cutout_on_the_left](https://github.com/material-components/material-components-android/assets/103059958/d05e662d-1667-4520-bff1-9a2022c8dba1)
![Google_Weather_Search_page__display_cutout_on_the_right](https://github.com/material-components/material-components-android/assets/103059958/a33aba53-cc59-41b2-b45f-b930ad824a72)
![Android settings display cutout issue in landscape mode](https://github.com/material-components/material-components-android/assets/103059958/5f65daf0-cf7c-4769-afd4-6ce4bcb5954f)
![Clock display cutout issue in landscape mode](https://github.com/material-components/material-components-android/assets/103059958/5fb4c112-6306-41c3-baee-865b5515a927)

**Comment by polodarb** [comment]
I agree, this greatly spoils the user experience when using applications
